pkgs/applications/editors/emacs/default.nix

```diff
@@ -31,7 +32,8 @@ stdenv.mkDerivation rec {
     sha256 = "0cwgyiyymnx4xdg99dm2drfxcyhy2jmyf0rkr9fwj9mwwf77kwhr";
   };
 
-  patches = lib.optional stdenv.isDarwin ./at-fdcwd.patch;
+  patches = lib.optional stdenv.isDarwin ./at-fdcwd.patch
+    ++ lib.optional withCsrc ./install-C-src.patch;
```

Comment:
@binarin, thanks for pointing this out. Is this documented somewhere or just general knowledge? Happy to change it.

The main advantage of doing it in the Makefile is to have access to the version number. I debated whether it was better to add the version number in the derivation or to do it this way.